mapper-processor/src/main/java/com/datastax/oss/driver/internal/mapper/processor/dao/DaoSetEntityMethodGenerator.java

```diff
@@ -117,15 +118,20 @@ public DaoSetEntityMethodGenerator(
     // Generate the method:
     String helperFieldName = enclosingClass.addEntityHelperField(ClassName.get(entityElement));
 
+    NullSavingStrategy nullSavingStrategy =
+        methodElement.getAnnotation(SetEntity.class).nullSavingStrategy();
+
     // Forward to the base injector in the helper:
     return Optional.of(
         GeneratedCodePatterns.override(methodElement)
             .addStatement(
-                "$L$L.set($L, $L)",
+                "$1L$2L.set($3L, $4L, $5T.$6L)",
```

Comment:
Nit: you don't strictly need to number placeholders if none of them is used more than once (but if you find it easier to read that's fine with me as well).

Reply:
Thx, I noticed that but I think that In this case having indices on them makes the code more readable.